lib/block-template-utils.php

```diff
@@ -98,6 +98,34 @@ function gutenberg_generate_block_templates_export_file() {
 	// Convert to a string.
 	$theme_json_encoded = wp_json_encode( $theme_json_raw, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE );
 
+	// Find any uploaded files.
+	$uploads = wp_upload_dir();
```

Comment:
This targets uploaded files only. Other images, such as files that point to external hosts should be left untouched.